### PR TITLE
Update Babel

### PR DESCRIPTION
**Pull request by chadhietala**
This commit removes esperato in favor of Babel. It currently uses a fork and is waiting for https://github.com/babel/broccoli-babel-transpiler/pull/32 to be merged.

### CODE REVIEW

**Comment by stefanpenner** [comment]
yay

File: tests/expectations/templates/application.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
-define('dummy/templates/application', ['exports', 'ember'], function (exports, Ember) {
+define("dummy/templates/application", ["exports", "ember"], function (exports, _ember) {
+  "use strict";
 
-	'use strict';
+  Object.defineProperty(exports, "__esModule", {
```
Comment (stefanpenner) [inline tests/expectations/templates/application.js]: prefer `amdStrict` which doesn't use the extra defProps

**Comment by chadhietala** [comment]
@stefanpenner I need to still update the dep-graph usage in the pre-packager as the format is pretty different.

**Comment by stefanpenner** [comment]
:+1:

**Pull request merged**